Components/Queues/Drivers/QueueDriverInterface.php

```diff
@@ -10,6 +10,7 @@ interface QueueDriverInterface extends SerializableInterface
     const FORMAT_JSON = 'json';
     const FORMAT_XML = 'xml';
     const DEFAULT_TTL = 86400;
+    const DEFAULT_PORT = 5672;
```

Comment:
This port is not a generic port, it is the port related to amqp and shouldn't be considered as DEFAULT port, in Stomp the port is 61613 FOR INSTANCE

Reply:
I need to specify a port to connect with the AMQP driver. It's one of the requirements of the connections. But, anyway I moved this constant to the AsyncQueueDriverInterface. There is also the possibility to pass other port by parameters on SmartboxIntegrationFrameworkExtension file.